### ISSUE TITLE
各位大佬好，我是一个萌新，想复现这个模型，但是在数据集输入进去之后，会报出这样的错误，是为什么呢？

### ISSUE THREAD

**Issue by DirectorZ1** [kwuking/TimeMixer#43]
usage: run.py [-h] --task_name TASK_NAME --is_training IS_TRAINING --model_id MODEL_ID --model MODEL --data DATA [--root_path ROOT_PATH]
              [--data_path DATA_PATH] [--features FEATURES] [--target TARGET] [--freq FREQ] [--checkpoints CHECKPOINTS] [--seq_len SEQ_LEN]
              [--label_len LABEL_LEN] [--pred_len PRED_LEN] [--seasonal_patterns SEASONAL_PATTERNS] [--inverse] [--top_k TOP_K]
              [--num_kernels NUM_KERNELS] [--enc_in ENC_IN] [--dec_in DEC_IN] [--c_out C_OUT] [--d_model D_MODEL] [--n_heads N_HEADS]
              [--e_layers E_LAYERS] [--d_layers D_LAYERS] [--d_ff D_FF] [--moving_avg MOVING_AVG] [--factor FACTOR] [--distil] [--dropout DROPOUT]     
              [--embed EMBED] [--activation ACTIVATION] [--output_attention] [--channel_independence CHANNEL_INDEPENDENCE]
              [--decomp_method DECOMP_METHOD] [--use_norm USE_NORM] [--down_sampling_layers DOWN_SAMPLING_LAYERS]
              [--down_sampling_window DOWN_SAMPLING_WINDOW] [--down_sampling_method DOWN_SAMPLING_METHOD]
              [--use_future_temporal_feature USE_FUTURE_TEMPORAL_FEATURE] [--num_workers NUM_WORKERS] [--itr ITR] [--train_epochs TRAIN_EPOCHS]        
              [--batch_size BATCH_SIZE] [--patience PATIENCE] [--learning_rate LEARNING_RATE] [--des DES] [--loss LOSS] [--lradj LRADJ]
              [--pct_start PCT_START] [--use_amp] [--comment COMMENT] [--use_gpu USE_GPU] [--gpu GPU] [--use_multi_gpu] [--devices DEVICES]
              [--p_hidden_dims P_HIDDEN_DIMS [P_HIDDEN_DIMS ...]] [--p_hidden_layers P_HIDDEN_LAYERS]
run.py: error: the following arguments are required: --task_name, --is_training, --model_id, --model, --data

**Comment by kwuking** [comment]
你好，这个报错日志显示的是缺少了--task_name, --is_training, --model_id, --model, --data这些参数，建议直接使用sh脚本运行。

**Comment by mumu-learn** [comment]
是由于parser.add_argument里面对应参数的required设置为了true就是需要参数，找到这几个改成False就好了
